src/api.rs

```diff
@@ -43,6 +43,18 @@ use crate::utils::sourcemaps::get_sourcemap_reference_from_headers;
 use crate::utils::ui::{capitalize_string, make_byte_progress_bar};
 use crate::utils::xcode::InfoPlist;
 
+const QUERY_ENCODE_SET: &AsciiSet = &CONTROLS.add(b' ').add(b'"').add(b'#').add(b'<').add(b'>');
+const DEFAULT_ENCODE_SET: &AsciiSet = &CONTROLS
```

Comment:
Could you add to `QUERY_ENCODE_SET` instead of repeating?

Reply:
Wasn't sure if it's the same reference or does builder always return new instance.